src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureDetails/FeatureDetails.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+    PlaygroundFeatureSchema,
+    PlaygroundRequestSchema,
+} from 'hooks/api/actions/usePlayground/playground.model';
+import { Alert, IconButton, Typography, useTheme } from '@mui/material';
+import { PlaygroundResultChip } from '../../PlaygroundResultChip/PlaygroundResultChip';
+import { useStyles } from './FeatureDetails.styles';
+import { CloseOutlined } from '@mui/icons-material';
+import React from 'react';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import {
+    checkForEmptyValues,
+    hasCustomStrategies,
+    hasOnlyCustomStrategies,
+} from './helpers';
+
+interface PlaygroundFeatureResultDetailsProps {
+    feature: PlaygroundFeatureSchema;
+    input?: PlaygroundRequestSchema;
+    onClose: () => void;
+}
+export const FeatureDetails = ({
+    feature,
+    input,
+    onClose,
+}: PlaygroundFeatureResultDetailsProps) => {
+    const { classes: styles } = useStyles();
+    const theme = useTheme();
+
+    const description = feature.isEnabled
+        ? `This feature toggle is True in ${input?.environment} because `
+        : `This feature toggle is False in ${input?.environment} because `;
+
+    const reason = (() => {
+        if (feature.isEnabled) return 'at least one strategy is True';
+
+        if (!feature.isEnabledInCurrentEnvironment)
+            return 'the environment is disabled';
+
+        if (hasOnlyCustomStrategies(feature))
+            return 'no strategies could be fully evaluated';
+
+        return 'all strategies are either False or could not be fully evaluated';
+    })();
+
+    const color = feature.isEnabled
+        ? theme.palette.success.main
+        : theme.palette.error.main;
+
+    const noValueTxt = checkForEmptyValues(input?.context)
+        ? 'You did not provide a value for your context field in step 2 of the configuration'
+        : undefined;
+
+    const customStrategiesTxt = hasCustomStrategies(feature)
+        ? `You have custom strategies. Custom strategies can't be evaluated and they will be set as Unevaluated`
```

Comment:
Repeating (almost) my text change suggestion from my previous reviews:
```suggestion
        ? `This feature uses custom strategies. Custom strategies can't be evaluated, so they will be marked as Unevaluated`
```